Samples/Sample.Server.WebAuthenticator/Controllers/MobileAuthController.cs

```diff
@@ -31,12 +31,16 @@ public async Task Get([FromRoute]string scheme)
             }
             else
             {
+               var claims = auth.Principal.Identities.FirstOrDefault()?.Claims;
+                    string email = string.Empty;
+                    email = claims?.FirstOrDefault(c => c.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress")?.Value;
```

Comment:
We should really use the constant for this:

`System.Security.Claims.ClaimTypes.Email` as seen here: https://docs.microsoft.com/en-us/dotnet/api/system.security.claims.claimtypes.email

Reply:
Agreed. Was not aware of the ClassTypes Constants